pkg/controller/component/component_lifecycle_action_utils.go

```diff
@@ -214,6 +214,9 @@ func renderActionCmdJob(ctx context.Context,
 		for i := range job.Spec.Template.Spec.Containers {
 			intctrlutil.InjectZeroResourcesLimitsIfEmpty(&job.Spec.Template.Spec.Containers[i])
 		}
+		if customAction.RetryPolicy != nil && customAction.RetryPolicy.MaxRetries > 0 {
+			job.Spec.BackoffLimit = func(i int32) *int32 { return &i }(int32(customAction.RetryPolicy.MaxRetries))
```

Comment:
can instead of the function `pointer.Int32`

Reply:
> can instead of the function `pointer.Int32`

fixed done